darwin/dataset/release.py

```diff
@@ -83,22 +184,26 @@ def download_zip(self, path: Path) -> Path:
         Returns
         --------
         Path
-            Same Path as provided in the parameters.
-        
+            Same ```Path``` as provided in the parameters.
```

Comment:
Should it use just 2 backticks?

<pre>
Same ``Path`` as provided in the parameters.
</pre>

Reply:
There are more of these which I won't mark not to clutter the review.